### DISCUSSION TITLE
Multipule people in an image when not requested

### DISCUSSION THREAD

**Discussion by AuroraDizon** [basujindal/stable-diffusion#165] [General]
I used the same prompt on the standard Stable Diffusion and have no issue. There seems to be something that causes it to generate 3-4 or more people when not asked to make that many in an image.

**Comment by TokisanGames** [comment]
Did you expand your width or height to say 1024? That will tend to produce 2-4 objects.

**Comment by AuroraDizon** [comment]
Yeah, I had the w and h up to 900. Alright thanks for the reply.

**Comment by TokisanGames** [comment]
The model is trained on 512, so can't go much beyond that with text input. However, if you want higher resolution, you can try forcing a composition with img2img plus your text.
